### PR TITLE
Add autoplay to homepage

### PR DESCRIPTION
**Pull request by floustao**
preview-autoplay-homepage

+ autoplay on homepage
+ add conditions in site.js to fire JS functions once user is on right page
+ my idea to gather all dot-related styling was not the greatest after all... I split each one in its specific component :)

### CODE REVIEW

File: source/javascripts/site.js [hunk 1]
```diff
@@ -8,8 +8,14 @@ import carouselAtelier from './components/page_atelier';
 
 mobileMenu();
 contactForm();
-changeBannerPicture();
-carouselAtelier(2);
+
+if (document.querySelector(".home-content")) {
```
Comment (grmnlrt) [inline source/javascripts/site.js]: 👍

File: source/javascripts/site.js [hunk 2]
```diff
@@ -8,8 +8,14 @@ import carouselAtelier from './components/page_atelier';
...
+  changeBannerPicture(2);
+}
+
+if (document.querySelector("#atelier-pictures")) {
```
Comment (grmnlrt) [inline source/javascripts/site.js]: 👍

**Comment by floustao** [comment]
Yes of course ! C'est mieux effectivement.
30px right et 30px bottom as below ?

image

**Comment by grmnlrt** [comment]
PAR-FAIT

**Pull request merged**